WordPress/Classes/ViewRelated/Me/App Settings/About/AutomatticAboutScreen.swift

```diff
@@ -62,7 +62,7 @@ class AutomatticAboutScreen: UIViewController {
 
         // Setting the frame once is needed so that the table view header will show.
         // This seems to be a table view bug although I'm not entirely sure.
-        headerView.frame.size = headerView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize)
+        headerView.frame.size.height = headerView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
```

Comment:
I agree that this feels a bit like a hack, but if that's the only way for the header to show..